src/Honeybee_ Run Energy Simulation.py

```diff
@@ -1223,20 +1226,10 @@ def writeBatchFile(self, workingDir, idfFileName, epwFileAddress, EPDirectory =
         batchfile = open(batchFileAddress, 'w')
         batchfile.write(batchStr)
         batchfile.close()
-    
```

Comment:
@antonszilasi again you're overwriting latest changes that I made to the component. Can you please get this fixed and send a new PR. Thanks.